tests/Feature/__snapshots__/BlockConverterTest__test_microsoft_word_importing__1.txt

```diff
@@ -0,0 +1,10 @@
+<!-- wp:paragraph --><p class="MsoNormal" style="margin-bottom:12.0pt;line-height:115%"><b><span style='font-size:11.0pt;line-height:115%;font-family:"Calibri",sans-serif;color:black'>This is a test from Microsoft Word</span></b></p><!-- /wp:paragraph -->
```

Comment:
I'd love to take this a step further and also get rid of the `class="MsoNormal"` and the inline style and rewrite `<b>` to `<strong>` and get rid of the span etc. What does WordPress do if you paste this content into Gutenberg? How far does it go? I'd love to have this library generate similar results to a "paste from Word" operation in Gutenberg in the browser. Is there a PHP library we can lean on to make this easier? Surely someone has solved this problem already?